types/underscore/underscore-tests.ts

```diff
@@ -474,6 +474,46 @@ _(explicitNumberDictionary).each((value, key, collection) => {
     _.chain(anyValue).forEach(anyCollectionIterator); // // $ExpectType _Chain<any, any>
 }
 
+// get
+
+{
+    // no default value
+    // $ExpectType number | undefined
+    _.get({ a: numberValue }, 'a');
+
+    // trying to get the property on a simple path
+    // $ExpectType number
+    _.get({ a: numberValue }, ['a'], numberValue);
+
+    // trying to get the property one level deep
+    // $ExpectType number | { b: number; }
+    _.get({ a: { b: numberValue } }, ['a'], numberValue);
+
+    // trying to get the property one level deep
+    // $ExpectType string | number | { b: number; }
+    _.get({ a: { b: numberValue }, c: "a" }, ['a', 'b'], numberValue);
```

Comment:
If you want more accurate typings for nested paths, you can try your hand at a combination of the `keyof` operator and one or two helper types similar to `ListItemOrSelf` (lines 143-153). I should mention that the PR becomes a lot more ambitious in that case, though.